forge-gui/res/cardsfolder/upcoming/biophagus.txt

```diff
@@ -3,9 +3,9 @@ ManaCost:1 G
 Types:Creature Human Tyranid Wizard
 PT:1/3
 A:AB$ Mana | Cost$ T | Produced$ Any | PrecostDesc$ Genomic Enhancement — | TriggersWhenSpent$ TrigSpent | SpellDescription$ Add one mana of any color. If this mana is spent to cast a creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
-SVar:TrigSpent:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | Execute$ TrigEffect | TriggerDescription$ If this mana is spent to cast a creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
+SVar:TrigSpent:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | Execute$ TrigEffect | Static$ True
 SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | ForgetOnMoved$ Stack | ReplacementEffects$ ETBCreat
 SVar:ETBCreat:Event$ Moved | ValidCard$ Card.IsRemembered | Destination$ Battlefield | ReplaceWith$ DBPutP1P1 | ReplacementResult$ Updated | Description$ That creature enters the battlefield with an additional +1/+1 counter on it and gains hexproof until your next turn.
```

Comment:
Description is wrong (feel weird to comment on my own card lol)

Reply:
The good news is that `Description$` shouldn't show up anywhere anyway